boardwalk/prod.yml

```diff
@@ -96,10 +99,9 @@ services:
       - server
     restart: always
   boardwalk:
-    image: quay.io/ucsc_cgl/commons-boardwalk:commons-0.1.1-beta
+    image: quay.io/ucsc_cgl/commons-boardwalk:commons-0.1.2-beta
     environment:
       - BOARDWALK_HOST=${dcc_dashboard_host}
-      - BW_DATA_URL=${dcc_dashboard_protocol}://${dcc_dashboard_host}
```

Comment:
This should not be showing up as change. Can you please resolve it so it doesn't.

Reply:
We decided not to worry about this as the branch merges with `feature/commons`.